src/external_app/external_auth.ts

```diff
@@ -81,7 +87,9 @@ export class ExternalAuth extends Auth {
     this._tokenCallbackPromise = new Promise<RefreshTokenResponse>(
       (resolve, reject) => {
         window[CALLBACK_SET_TOKEN] = (success, data) =>
-          success ? resolve(data) : reject(data);
+          success
+            ? resolve(data)
+            : reject(new Error("Refresh token unsuccessful; "));
```

Comment:
Why was this changed? We still want to use the data we get from the external app?

Reply:
Yes your are right. I did not think that the data maybe still useful on a reject. I reverted the change. Thanks for the review 👍